channels/iq.m3u

```diff
@@ -1,49 +1,51 @@
 #EXTM3U x-tvg-url="http://195.154.221.171/epg/guidearab.xml.gz"
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="General",ABNsat
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="http://vidmgr.abnvideos.com/images/playlists/phpENpJQS.jpeg" group-title="General",ABNsat
 http://rtmp1.abnsat.com/hls/arabic.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="General",Al Fady
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/wyy5unY.jpg" group-title="General",Al Fady
 http://media.smc-host.com:1935/redirect/alfadytv.tv/alfadytv.smil?type=m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="General",Al Fallujah
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="http://www.alfallujah.tv/sites/all/themes/progressive/progressive_sub/logo.png" group-title="General",Al Fallujah
 https://player-api.new.livestream.com/accounts/4671269/events/4213685/live.m3u8
-#EXTINF:-1 tvg-id="Al Hurra Iraq ARB" tvg-name="Al Hurra Iraq ARB" tvg-language="Arabic" tvg-logo="" group-title="News",Al Hurra Iraq
+#EXTINF:-1 tvg-id="Al Hurra Iraq ARB" tvg-name="Al Hurra Iraq ARB" tvg-language="Arabic" tvg-logo="https://www.alhurra.com/Content/responsive/MBN/ar-ALH/img/logo.svg" group-title="News",Al Hurra Iraq
 http://mbnhls-lh.akamaihd.net/i/MBN_1@118619/master.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="General",Al Masara Al Oula
+#EXTINF:-1 tvg-id="" tvg-name="Al-Masar Al-Oula" tvg-language="Arabic" tvg-logo="https://i.imgur.com/npuXX8e.jpg" group-title="General",Al-Masar Al-Oula
```

Comment:
tvg-name is wrong. It should be the same as the display-name from the EPG. If there is no such channel in the EPG, it is better to leave this attribute blank.